src/renderer/dx/CustomTextLayout.h

```diff
@@ -90,7 +90,7 @@ namespace Microsoft::Console::Render
             UINT8 bidiLevel;
             bool isNumberSubstituted;
             bool isSideways;
-            ::Microsoft::WRL::ComPtr<IDWriteFontFace5> fontFace;
+            ::Microsoft::WRL::ComPtr<IDWriteFontFace1> fontFace;

```

Comment:
For COM interfaces I'm not sure the best way to validate that things still work on win7 since just building with the right `WINVER` value won't cause a build error. If there were windows 7 azure machines we could set up a CI to run automated tests there but it doesn't look like those exist.

Reply:
How about at the least, can you leave a block comment at the top of the file with some of the information/warnings?

I thought about a compiler #warning or #error but it would probably otherwise block compilation.

I would just like some sort of reference material that says
// To maintain compatibility with Windows 7, we have to ensure that pointers we hold don't go beyond what was available in Windows 7 without testing OS version.
// You can find which versions of the interfaces were available in each OS version at:
// Windows 7: https://example.com/windows7dxapis
// Windows 8: https://example.com/windows8dxapis
// Windows 8.1: https://example.com/windows81dxapis

Or wherever you got the information from to decide which version helper to use and where the boundary was.